### PR TITLE
Switch assessment out for pre-award-frontend

### PR DESCRIPTION
**Pull request by samuelhwilliams**
The assessment frontend code now lives in pre-award-frontend, so we
don't need to run a separate app locally for it anymore.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -136,18 +109,22 @@ services:
...
+      - ASSESS_HOST=assessment.levellingup.gov.localhost:3010
       - FLASK_DEBUG=1
       - FLASK_ENV=development
       - REDIS_INSTANCE_URI=redis://redis-data:6379
       - SECRET_KEY=dc_key
+      - ASSESSMENT_STORE_API_HOST=http://pre-award-stores:8080/assessment
     env_file: .awslocal.env
     ports:
       - 3008:8080
+      - 3010:8080
```
Comment (MarcUsher) [inline docker-compose.yml:121]: Ooh interesting. Any reason we're serving this up on a separate port where we didn't for the post-award consolidation?

**Review by MarcUsher — APPROVED** [review]
Looks good! Just one little question for my own education.

We can probably also remove `assessment` from the list of repos in the venv/repo reset scripts?

File: docker-compose.yml (hunk 1 above)
Comment (samuelhwilliams) [reply docker-compose.yml:121]: Mostly just so that people don't have to update all of their local URLs 😂

Comment (MarcUsher) [reply docker-compose.yml:121]: Haha cool, that's a good enough reason to me 😄

**Pull request merged**